src/marqo/tensor_search/tensor_search.py

```diff
@@ -2597,6 +2562,18 @@ def _create_score_modifiers_tensor_search_query(result_count, offset, vector_fie
     }
     return search_query
 
+
+def delete_documents(config: Config, index_name: str, doc_ids: List[str], auto_refresh):
+    """Delete documents from the Marqo index with the given doc_ids """
+    return delete_docs.delete_documents(
```

Comment:
When we go about an entire reformatting of tensor_search/, it may be worth having `format_delete_docs_response` in this function instead. Therefore operation specific files deal in Pydantic objects, and the tensor_search.py operations are responsibly for mapping args -> RequestObject, then ResponseObject -> Dict

Reply:
Or have the decoding within the pydantic model (i.e. controlled via the `.json` method).